.gitignore

```diff
@@ -41,3 +41,8 @@ Splunk_TA_Dynatrace_ucc/*
 demo_addon_for_splunk/
 demo_addon_for_splunk_already_exists/
 init_addon_for_ucc_package/
+
+# UI build
+# ignore everything except redirect_page.js
```

Comment:
Please include this into poetry.include and we are good to go otherwise we will miss those files after poetry build.